sdk/eventhub/Azure.Messaging.EventHubs.Shared/tests/Azure.Messaging.EventHubs.Shared.Tests.csproj

```diff
@@ -27,6 +31,7 @@
   <ItemGroup>
     <ProjectReference Include="../../Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj" />
     <ProjectReference Include="$(AzureCoreTestFramework)" />
+    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

Comment:
nit:  Do we need the explicit reference?   

_(I'm not sure why we've got one at all, honestly.  We should pick it up transiently via the Event Hubs project reference.  Whatever reason that we had, I can't remember it.)_